gef.py

```diff
@@ -9810,12 +9812,15 @@ def set_setting(self, argv: Tuple[str, Any]) -> None:
                 _newval = True if new_value.upper() in ("TRUE", "T", "1") else False
             else:
                 _newval = new_value
-
-            gef.config[key] = _newval
         except Exception as e:
             err(f"'{key}' expects type '{_type.__name__}', got {type(new_value).__name__}: reason {str(e)}")
             return
 
+        try:
```

Comment:
If you want to move this block, might as well remove the try/except above, because nothing will throw

Reply:
Oh. How was it we got the exception raised when we couldn't cast the value to the expected type? I wanted a separate error for that.

Reply:
Understood, never mind then